homeassistant/components/switch/rachio.py

```diff
@@ -30,6 +30,7 @@
 
 
 # noinspection PyUnusedLocal
+# noinspection PyUnresolvedLocal
```

Comment:
Please remove these warnings. They are not part of linters that the project uses.